### PR TITLE
Updated exit nodes.

### PR DESCRIPTION
**Pull request by jbostrus**
LA tcp removed - not listed in xml / hasn't been working for a while
LA2 udp removed - not listed in xml / hasn't been working for a while
Added freemium endpoints
Changed (rather than adding) a couple other endpoints that had different ip's listed in xml where old ip's didn't seem to work any longer.
Changed 2 existing entries IP's to reflect current xml listing.

### CODE REVIEW

File: proxpn [hunk 1]
```diff
@@ -136,21 +136,28 @@ TCP_EXIT_NODES[SanJose]=179.48.248.104
...
 UDP_EXIT_NODES[NYC]=196.52.16.2
+UDP_EXIT_NODES[NYC2]=196.52.17.68
 UDP_EXIT_NODES[Miami]=196.52.22.2
+UDP_EXIT_NODES[Miami2]=196.52.23.66
 UDP_EXIT_NODES[Seattle]=196.52.18.2
+UDP_EXIT_NODES[Seattle2]=196.52.19.241
 UDP_EXIT_NODES[Chicago]=50.7.1.67
+UDP_EXIT_NODES[Chicago2]=196.52.21.225
 UDP_EXIT_NODES[Toronto]=162.253.128.67
 UDP_EXIT_NODES[HongKong]=43.251.159.25
```
Comment (jbostrus) [inline proxpn:146]: HongKong listings are listed in XML but could not connect from USA.

File: proxpn [hunk 2]
```diff
@@ -136,21 +136,28 @@ TCP_EXIT_NODES[SanJose]=179.48.248.104
...
-UDP_EXIT_NODES[LA]=64.27.29.74
-UDP_EXIT_NODES[LA2]=64.27.29.70
+UDP_EXIT_NODES[HongKong2]=103.194.43.131
+UDP_EXIT_NODES[HongKong3]=103.194.43.26
+UDP_EXIT_NODES[LA]=188.172.214.196
+#UDP_EXIT_NODES[LA2]=64.27.29.70
 UDP_EXIT_NODES[Netherlands]=213.179.212.2
 UDP_EXIT_NODES[Netherlands2]=213.179.208.146
-UDP_EXIT_NODES[Stockholm]=94.185.84.48
+UDP_EXIT_NODES[Stockholm]=94.185.84.42
```
Comment (jbostrus) [inline proxpn:152]: Neither old nor new Stockholm IP seems to work from USA. Could just temporarily be down. New IP is current listed one in xml as of 2016/05/31 00:30 Pacific Time.

**Comment by MattSurabian** [comment]
Thanks! I'll review today and cut a new release barring any issues.

**Comment by MattSurabian** [comment]
Sorry for the delay! Reviewing now

**Comment by MattSurabian** [comment]
:+1:

File: proxpn (hunk 1 above)
Comment (Pysis868) [inline proxpn:146]: That's good you left the Hong Kong ones in there, in case they work for someone else.  Don't want to lose information :).

Comment (MattSurabian) [inline proxpn:146]: Yeah so long as the IPs are right and they're in the exit nodes manifest from ProXPN we should do our best to support them.

**Pull request merged**